### PR TITLE
Only one ParaId variable in the Parachain Template

### PR DESCRIPTION
**Pull request by shawntabrizi**
Many problems can occur when building and testing a Parachain caused by misconfiguring the paraid.

This can happen when there are 3 different places you need to update!

This PR makes it so a SINGLE location is the source of truth for the ParaId.

### CODE REVIEW

**Review by skunert — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]
cc @iulianbarbu

File: templates/parachain/node/src/chain_spec.rs [hunk 1]
```diff
@@ -35,11 +35,7 @@ pub fn development_chain_spec() -> ChainSpec {
 
 	ChainSpec::builder(
 		runtime::WASM_BINARY.expect("WASM binary was not built, please build it!"),
-		Extensions {
-			relay_chain: "rococo-local".into(),
-			// You MUST set this to the correct network!
-			para_id: 1000,
-		},
+		Extensions { relay_chain: "rococo-local".into(), para_id: runtime::PARACHAIN_ID },
```
Comment (iulianbarbu) [inline templates/parachain/node/src/chain_spec.rs:38]: nit: can we do the same thing for `rococo-local`?

**Review by iulianbarbu — APPROVED** [review]

**Review by pepoviola — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Pull request merged**